front/src/components/header/Header.tsx

```diff
@@ -140,9 +141,19 @@ const Header = ({ isLoggedIn, setIsLoggedOut }: Props) => {
           <NavLink
             isActiveRoute={activeRoute === '/my-shows'}
             linkTo="/my-shows"
-            mobileWidth="84px"
+            mobileWidth="100px"
             text="My Shows"
           />
+          {isLoggedIn ? (
+            <Box display={{ xs: 'block', md: 'none' }}>
+              <NavLink
+                isActiveRoute={activeRoute === '/my-profile'}
```

Comment:
my-profile should be settings

Reply:
@Jaskaran121 I think this is last place that needs to be updated, then we can merge!